WinUIGallery/DataModel/ControlInfoDataSchema.json

```diff
@@ -0,0 +1,132 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+        "Groups": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "UniqueId": {
+                        "type": "string",
+                        "description": "Globally unique id of this group",
+                        "uniqueItems": true
+                    },
+                    "Title": {
+                        "type": "string",
```

Comment:
I would probably write something like "The name of the control group. Will be displayed on the control group page and navigation".

Same for the other descriptions, I think ideally we should tell the intention of the property and then tell where it is being used.